### DISCUSSION TITLE
Can I export data as a CSV file?

### DISCUSSION THREAD

**Discussion by GlenWarren** [orchidsoftware/platform#2175] [Q&A]
I have tried a number of methods to export data as a CSV File, but I am not getting anywhere.

Example:
```
        // Set the content type
        header('Content-type: application/csv');

        // Set the file name option to a filename of your choice.
        header("Content-Disposition: attachment; filename=$filename");

        // Set the encoding
        header("Content-Transfer-Encoding: UTF-8");

        $file = fopen('php://output', 'w+');
        
        fputcsv($file, ['hello,bonjour', 'goodbye,adios']);

        // Close the file
        fclose($file);
```

**Comment by tabuna** [comment]
Hi @GlenWarren The base installation already has an export example. You can see its implementation:

https://github.com/orchidsoftware/platform/blob/b4abbb4434c28b7020078b95a18f75e67943c989/stubs/app/Orchid/Screens/Examples/ExampleScreen.php#L204-L225
